sshagent_windows.go

```diff
@@ -0,0 +1,35 @@
+//go:build windows
+
+package main
+
+import (
+	"github.com/davidmz/go-pageant"
+	"github.com/reconquest/hierr-go"
+
+	"github.com/Microsoft/go-winio"
+	"golang.org/x/crypto/ssh/agent"
+)
+
+const (
+	openSshAgentPipe = `\\.\pipe\openssh-ssh-agent`
+)
+
+func getSshAgent() (agent.Agent, error) {
+	pageant_window := pageant.Available()
```

Comment:
The use of snake_case is not idiomatic in Go and it's highly recommended to use camelCase instead.

In this case you could even avoid creating a variable and just check it like `if pageant.Available() { ... } else { ... }`

Reply:
Ah, right.. that was a copy/paste.